### PR TITLE
Set default Switch valuee based on server settings

### PR DESCRIPTION
**Pull request by sk7**

### CODE REVIEW

File: src/components/new-conversation-dialog/create-room/index.js [hunk 1]
```diff
@@ -120,26 +121,28 @@ const CreateRoom = ({
...
-          onChange={setIsPrivate}
-          className={classes.switch}
-          helpText={formatMessage({
-            id: 'ncdGroupPrivateHelpText',
-            defaultMessage:
-              'Only group members can view a private group and invite other people to join.',
-            description: 'switch help text',
-          })}
-        />
+        {org.defaults.groupDefaults.visibility !== 'private' && (
```
Comment (sk7) [inline src/components/new-conversation-dialog/create-room/index.js:124]: groupDefaults or groupDefaults.visibility might not be set.
we could use the `?.` operator to gracefully handle this  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Comment (sk7) [reply src/components/new-conversation-dialog/create-room/index.js:124]: ok seems like this value is always returned from the server but I will still change this to be sure

Comment (AleshaOleg) [reply src/components/new-conversation-dialog/create-room/index.js:124]: you can use `isPrivate` from props to toggle switch

Comment (AleshaOleg) [reply src/components/new-conversation-dialog/create-room/index.js:124]: Also according to issue, I think it's incorrect to hide switch completely

Comment (sk7) [reply src/components/new-conversation-dialog/create-room/index.js]: > Also according to issue, I think it's incorrect to hide switch completely

thx! fixed!

**Comment by AleshaOleg** [comment]
Value in state machine still will be false (public) even if switch is toggled to true (private)

**Pull request merged**